tools/buildHeaders/bin/makeHeaders

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+sh bin/makeExtinstHeaders
```

Comment:
Well, this bash partially answers the previous question.  Let's say my question is still valid, would like an answer, but not necessarily asking for a change, just the answer.

Reply:
Fixed